### ISSUE TITLE
 单机单卡可以运行吗

### ISSUE THREAD

**Issue by wuwangabc** [NiccoloCavagnero/PEM#16]
单机单卡可以运行吗

**Comment by Gabrysse** [comment]
Hi @wuwangabc, can you please post your problem in English? Thanks :)

**Comment by wuwangabc** [comment]
ok, Can it run on a single machine with a single graphics card?

**Comment by Gabrysse** [comment]
Yes, you can! You probably need to convert the SyncBN in normal BatchNorm operation. You can use the convert_sync_batchnorm function provided by the PyTorch library ([docs](https://pytorch.org/docs/stable/generated/torch.nn.SyncBatchNorm.html)).
You will also have to adjust some configuration (e.g. the lr and the batch size) and set the `--num-gpus` parameter to 1.

**Comment by wuwangabc** [comment]
Okay, thank you for your advice.
